data/en/2020/hispano/sponsors.yml

```diff
@@ -19,3 +19,7 @@ items:
     image: /images/sponsors/nicaragua_jug.png
     url: https://www.javanicaragua.org/
     width: 200px
+  - title: JUG Guatemala
+    image: /images/sponsors/guatemala_jug.png
+    url: https://www.guate-jug.net/
+    width: 100px
```

Comment:
I used 100px instead of the 200x others logo have mainly because the GuateJUG logo occupy the entire image frame so during my local test the 100 px looks better in relationship with the rest of existing logos size.